cas-server-core-tickets/src/test/java/org/jasig/cas/MockOnlyOneTicketRegistry.java

```diff
@@ -40,9 +40,9 @@ public Ticket getTicket(final String ticketId) {
     }
 
     @Override
-    public boolean deleteTicket(final String ticketId) {
+    public int deleteTicket(final String ticketId) {
         this.ticket = null;
-        return false;
+        return 0;
```

Comment:
I guess it's not a big deal with a mocked ticket registry. But shouldn't it return 1?